Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ScreenData.purs

```diff
@@ -225,6 +225,7 @@ initData = {
     , showDisabilityPopUp : false
     , isChatNotificationDismissed : false
     , searchLocationModelProps : dummySearchLocationModelProps
+    , flowWithoutOffers : true
```

Comment:
@S-PRAVINKUMAR are we using the `FLOW_WITHOUT_OFFERS` variable in order to show the initially selected radio button.

Reply:
If not can we update this value with the one stored in local store while first entering into this flow.